### ISSUE TITLE
Installation command failure

### ISSUE THREAD

**Issue by flaviu22** [microsoft/Bringing-Old-Photos-Back-to-Life#24]
Sorry for this trivial question. I have tried to run every command for installation:

```
cd Face_Enhancement/models/networks/
git clone https://github.com/vacancy/Synchronized-BatchNorm-PyTorch
cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .

```
at the step:

`cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
`
I got the following error:
_'cp' is not recognized as an internal or external command, operable program or batch file._

What I have done wrong ?

**Comment by mewbot97** [comment]
This project has Linux as requirement. I dont know if it will run in windows, however you can try using copy instead of cp for Windows.

**Comment by ModnarUser** [comment]
I managed to get it running on my Win10 machine with the help of [git bash](https://gitforwindows.org/),  Windows power shell and [chocolatey](https://chocolatey.org/) as package manager. 

`wget` had to be installed manually via `choco install wget` before running the corresponding `wget`-commands within the installation.

**Comment by flaviu22** [comment]
> This project has Linux as requirement. I dont know if it will run in windows, however you can try using copy instead of cp for Windows.

_cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm ._

You mean, copy **sync_batchnorm** folder where ? At one level upper ?

**Comment by flaviu22** [comment]
> This project has Linux as requirement. I dont know if it will run in windows, however you can try using copy instead of cp for Windows.

According with this documentation: https://www.computerhope.com/unix/ucp.htm

**Synchronized-BatchNorm-PyTorch/sync_batchnorm** folder should be copied in the same folder ? Or upper folder ?

**Comment by ModnarUser** [comment]
I don't quite understand your issue. Reguarding your question on where to copy the folder:
```bash
cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .
```
Will copy the `sync_batchnorm` folder and all subdirectories into the `Face_Enhancement/models/networks/` directory, which is where you would have to copy it manually aswell.
